docs/src/modules/java/pages/kickstart.adoc

```diff
@@ -0,0 +1,239 @@
+= Kickstart a Maven project
+
+
+include::ROOT:partial$include.adoc[]
+include::partial$attributes.adoc[]
+
+The Akka Serverless code generation tools for Java help you to get started quickly. They include:
+
+* A Maven archetype that generates the recommended project structure (including a `pom.xml`), a simple Counter service containing a Value Entity, and unit tests. A `README.md` explains what was created and how to work with the newly created service.
+
+* A Maven plugin that runs the gRPC compiler and generates code stubs. You can modify the `.proto` and source files, and the Akka Serverless Maven plugin will not overwrite your work but will generate code stubs for the elements you changed or added.
+
+The generated project also contains configuration for packaging and deploying the service.
+
+== Prerequisites
+
+Before running the code generation tools, make sure you have the following:
+
+* JDK {minimum-java-version} or higher (JDK {recommended-java-version} is recommended)
+* Apache Maven 3.6 or later
+* Docker {minimum_docker_version} or higher (to run locally)
+
+To deploy the generated service, you will need:
+
+* An https://developer.lightbend.com/docs/akka-serverless/getting-started/lightbend-account.html[Akka Serverless account]
+* An https://developer.lightbend.com/docs/akka-serverless/getting-started/projects.html#_create_a_project[Akka Serverless project]
+* The https://developer.lightbend.com/docs/akka-serverless/getting-started/set-up-development-env.html#_akka_serverless_cli[akkasls] CLI
+* A configured registry in which to publish the service container image. Refer to https://developer.lightbend.com/docs/akka-serverless/deploying/registries.html[Configuring registries] for more information on how to make your Docker registry available to Akka Serverless.
+
+:sectnums:
+== Configure Maven
+
+Lightbend hosts the archetype in a Maven repository. You need to provide configuration that points to the Lightbend repository in one of the following ways:
+
+* In a `settings.xml` file located in your home directory in a sub-directory named `.m2`. If you have an existing `settings.xml` file, either in your home directory or in `\{MAVEN_HOME}/conf`, add the `akkaserverless` profile element shown in the full example below and list it as `activeProfile` in the existing file.
+* Your organization may have configured a proxy to be used to download Maven artifacts, such as Jfrog Artifactory. If you have difficulties updating `settings.xml` for a proxy, please consult your system administrator.
+
+The following shows a complete `settings.xml` file configured to use the Lightbend repository:
+
+[source,xml]
+----
+include::example$settings.xml[]
+----
```

Comment:
If #72 gets merged first, we can already remove that section from the docs.

Reply:
Sounds good. We'll end up publishing these docs to make the switch over to the sources being here, so we may want to release the new archetypes first.